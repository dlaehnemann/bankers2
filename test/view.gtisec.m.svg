<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg width="3000px" height="3000px" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
<g id="bg">
<rect x="0" y="0" width="3000px" height="3000px" style="fill:rgb(255,255,255);"/>
</g>
<g id="ideograms">
<path d="M1500.000,250.000 A1250.000,1250.000 40.000 0,1 2618.621,2057.842 L2708.110,2102.469 A1350.000,1350.000 0.000 0,0 1500.000,150.000 Z" style="stroke-width:1.0;stroke:rgb(189,189,189);stroke-linecap:round;fill:none;" />
<path d="M1500.000,250.000 A1250.000,1250.000 40.000 0,1 2673.890,1070.485 L2767.801,1036.124 A1350.000,1350.000 0.000 0,0 1500.000,150.000 Z" style="stroke-linecap:round;fill:rgb(35,132,67);" />
<path d="M2673.890,1070.485 A1250.000,1250.000 40.000 0,1 2721.167,1233.071 L2818.860,1211.716 A1350.000,1350.000 0.000 0,0 2767.801,1036.124 Z" style="stroke-linecap:round;fill:rgb(0,104,55);" />
<path d="M2721.167,1233.071 A1250.000,1250.000 40.000 0,1 2746.038,1400.554 L2845.721,1392.598 A1350.000,1350.000 0.000 0,0 2818.860,1211.716 Z" style="stroke-linecap:round;fill:rgb(0,69,41);" />
<path d="M2746.038,1400.554 A1250.000,1250.000 40.000 0,1 2618.621,2057.842 L2708.110,2102.469 A1350.000,1350.000 0.000 0,0 2845.721,1392.598 Z" style="stroke-linecap:round;fill:rgb(165,0,38);fill-opacity:0.33;" />
<text x="2727.7" y="740.3" font-size="46.8px" font-family="CMUSansSerif-DemiCondensed" style="text-anchor:middle;fill:rgb(0,0,0);" transform="rotate(418.3,2727.7,740.3)" >NA00003</text>
<line x1='1500.0' y1='150.0' x2='1500.0' y2='130.0' style="stroke-width:3.0;stroke:rgb(0,0,0);stroke-linecap:round;fill:none;" />
<text x="1512.0" y="125.1" font-size="31.2px" font-family="CMUSansSerif" style="text-anchor:start;fill:rgb(0,0,0);" transform="rotate(270.0,1512.0,125.1)" >0 K</text>
<path d="M1500.000,250.000 A1250.000,1250.000 40.000 0,1 2618.621,2057.842 L2708.110,2102.469 A1350.000,1350.000 0.000 0,0 1500.000,150.000 Z" style="stroke-width:1.0;stroke:rgb(189,189,189);stroke-linecap:round;fill:none;" />
<path d="M2582.532,2125.000 A1250.000,1250.000 40.000 0,1 457.584,2189.833 L374.191,2245.020 A1350.000,1350.000 0.000 0,0 2669.134,2175.000 Z" style="stroke-width:1.0;stroke:rgb(189,189,189);stroke-linecap:round;fill:none;" />
<path d="M2582.532,2125.000 A1250.000,1250.000 40.000 0,1 2104.715,2593.993 L2153.092,2681.512 A1350.000,1350.000 0.000 0,0 2669.134,2175.000 Z" style="stroke-linecap:round;fill:rgb(35,132,67);" />
<path d="M2104.715,2593.993 A1250.000,1250.000 40.000 0,1 680.406,2443.804 L614.838,2519.308 A1350.000,1350.000 0.000 0,0 2153.092,2681.512 Z" style="stroke-linecap:round;fill:rgb(0,104,55);" />
<path d="M680.406,2443.804 A1250.000,1250.000 40.000 0,1 560.375,2324.381 L485.205,2390.332 A1350.000,1350.000 0.000 0,0 614.838,2519.308 Z" style="stroke-linecap:round;fill:rgb(0,69,41);" />
<path d="M560.375,2324.381 A1250.000,1250.000 40.000 0,1 457.584,2189.833 L374.191,2245.020 A1350.000,1350.000 0.000 0,0 485.205,2390.332 Z" style="stroke-linecap:round;fill:rgb(165,0,38);fill-opacity:0.33;" />
<text x="1545.1" y="2979.1" font-size="46.8px" font-family="CMUSansSerif-DemiCondensed" style="text-anchor:middle;fill:rgb(0,0,0);" transform="rotate(-1.7,1545.1,2979.1)" >NA00002</text>
<line x1='2669.1' y1='2175.0' x2='2686.5' y2='2185.0' style="stroke-width:3.0;stroke:rgb(0,0,0);stroke-linecap:round;fill:none;" />
<text x="2684.7" y="2197.9" font-size="31.2px" font-family="CMUSansSerif" style="text-anchor:start;fill:rgb(0,0,0);" transform="rotate(30.0,2684.7,2197.9)" >0 K</text>
<path d="M2582.532,2125.000 A1250.000,1250.000 40.000 0,1 457.584,2189.833 L374.191,2245.020 A1350.000,1350.000 0.000 0,0 2669.134,2175.000 Z" style="stroke-width:1.0;stroke:rgb(189,189,189);stroke-linecap:round;fill:none;" />
<path d="M417.468,2125.000 A1250.000,1250.000 40.000 0,1 1423.795,252.325 L1417.699,152.511 A1350.000,1350.000 0.000 0,0 330.866,2175.000 Z" style="stroke-width:1.0;stroke:rgb(189,189,189);stroke-linecap:round;fill:none;" />
<path d="M417.468,2125.000 A1250.000,1250.000 40.000 0,1 264.831,1308.014 L166.018,1292.655 A1350.000,1350.000 0.000 0,0 330.866,2175.000 Z" style="stroke-linecap:round;fill:rgb(35,132,67);" />
<path d="M264.831,1308.014 A1250.000,1250.000 40.000 0,1 1092.439,318.309 L1059.834,223.774 A1350.000,1350.000 0.000 0,0 166.018,1292.655 Z" style="stroke-linecap:round;fill:rgb(0,104,55);" />
<path d="M1092.439,318.309 A1250.000,1250.000 40.000 0,1 1255.877,274.070 L1236.348,175.996 A1350.000,1350.000 0.000 0,0 1059.834,223.774 Z" style="stroke-linecap:round;fill:rgb(0,69,41);" />
<path d="M1255.877,274.070 A1250.000,1250.000 40.000 0,1 1423.795,252.325 L1417.699,152.511 A1350.000,1350.000 0.000 0,0 1236.348,175.996 Z" style="stroke-linecap:round;fill:rgb(165,0,38);fill-opacity:0.33;" />
<text x="228.2" y="816.6" font-size="46.8px" font-family="CMUSansSerif-DemiCondensed" style="text-anchor:middle;fill:rgb(0,0,0);" transform="rotate(-61.7,228.2,816.6)" >NA00001</text>
<line x1='330.9' y1='2175.0' x2='313.5' y2='2185.0' style="stroke-width:3.0;stroke:rgb(0,0,0);stroke-linecap:round;fill:none;" />
<text x="315.3" y="2197.9" font-size="31.2px" font-family="CMUSansSerif" style="text-anchor:end;fill:rgb(0,0,0);" transform="rotate(-30.0,315.3,2197.9)" >0 K</text>
<path d="M417.468,2125.000 A1250.000,1250.000 40.000 0,1 1423.795,252.325 L1417.699,152.511 A1350.000,1350.000 0.000 0,0 330.866,2175.000 Z" style="stroke-width:1.0;stroke:rgb(189,189,189);stroke-linecap:round;fill:none;" />
</g>
<g id="track_1">
<path d="M 1096.515,330.126 A 1237.500,1237.500 0.00 0,1 1258.3,286.3 L 1258.3,286.3 1274.2,369.4 1288.1,448.8 1300.2,524.7 1310.5,597.4 1319.1,667.0 1326.0,733.7 1331.5,797.7 1335.4,859.1 1337.9,918.3 1339.1,975.3 1338.9,1030.3 1337.3,1083.5 1334.5,1135.1 1330.4,1185.3 1325.0,1234.2 1318.4,1282.0 1310.4,1328.8 1301.2,1374.9 1290.6,1420.4 1278.7,1465.4 1265.3,1510.1 1250.5,1554.7 1234.2,1599.4 1216.3,1644.2 1196.7,1689.3 1175.4,1734.9 1152.3,1781.1 1127.2,1828.1 1100.2,1876.0 1070.9,1924.9 1039.4,1975.1 1005.5,2026.5 969.1,2079.4 930.0,2133.8 888.1,2190.0 843.1,2248.0 795.0,2308.0 743.6,2370.1 688.6,2434.4  A 1237.500,1237.500 0.00 0,1 569.8,2316.1 L 569.8,2316.1 630.0,2262.3 686.4,2210.0 739.2,2158.9 788.5,2109.1 834.7,2060.4 877.7,2012.8 917.7,1966.1 955.0,1920.4 989.5,1875.4 1021.5,1831.1 1051.1,1787.5 1078.4,1744.3 1103.4,1701.4 1126.2,1658.9 1147.1,1616.5 1165.9,1574.2 1182.8,1531.9 1197.8,1489.3 1211.1,1446.5 1222.5,1403.3 1232.3,1359.5 1240.3,1315.1 1246.6,1270.0 1251.2,1223.9 1254.1,1176.8 1255.3,1128.5 1254.8,1078.9 1252.6,1027.9 1248.6,975.3 1242.7,920.9 1235.0,864.8 1225.3,806.6 1213.6,746.2 1199.8,683.5 1183.9,618.4 1165.7,550.7 1145.2,480.1 1122.1,406.7 1096.5,330.1  Z" style="stroke-width: 1.0;  stroke: rgb(115,115,115); fill: rgb(0,69,41); opacity: 0.167;" />
<path d="M 1096.515,330.126 A 1237.500,1237.500 0.00 0,1 1258.3,286.3 L 1258.3,286.3 1273.9,360.2 1290.2,430.3 1307.4,496.6 1325.4,559.3 1344.2,618.7 1363.8,674.8 1384.2,727.7 1405.5,777.7 1427.5,824.8 1450.4,869.1 1474.1,910.8 1498.8,950.0 1524.3,986.7 1550.8,1021.0 1578.2,1053.1 1606.7,1082.9 1636.3,1110.7 1667.0,1136.3 1698.9,1160.0 1732.0,1181.6 1766.4,1201.4 1802.2,1219.3 1839.5,1235.3 1878.2,1249.4 1918.6,1261.8 1960.7,1272.3 2004.5,1281.0 2050.3,1287.9 2098.0,1293.0 2147.8,1296.2 2199.8,1297.6 2254.0,1297.0 2310.8,1294.5 2370.0,1290.0 2431.9,1283.5 2496.6,1274.9 2564.3,1264.1 2635.0,1251.1 2709.0,1235.7  A 1237.500,1237.500 0.00 0,1 2733.6,1401.5 L 2733.6,1401.5 2654.9,1407.1 2579.7,1410.7 2508.0,1412.7 2439.5,1413.0 2374.0,1411.7 2311.5,1408.8 2251.7,1404.4 2194.5,1398.5 2139.7,1391.1 2087.2,1382.3 2036.9,1372.1 1988.6,1360.5 1942.3,1347.4 1897.7,1332.8 1854.7,1316.9 1813.4,1299.4 1773.4,1280.5 1734.8,1260.0 1697.4,1237.9 1661.2,1214.2 1626.0,1188.8 1591.7,1161.6 1558.4,1132.6 1525.8,1101.6 1494.0,1068.7 1462.9,1033.6 1432.3,996.4 1402.3,956.8 1372.8,914.8 1343.7,870.2 1315.1,823.0 1286.8,772.9 1258.8,719.8 1231.1,663.7 1203.7,604.2 1176.6,541.3 1149.7,474.8 1123.0,404.5 1096.5,330.1  Z" style="stroke-width: 1.0;  stroke: rgb(115,115,115); fill: rgb(0,69,41); opacity: 0.167;" />
<path d="M 688.602,2434.366 A 1237.500,1237.500 0.00 0,1 569.8,2316.1 L 569.8,2316.1 638.2,2256.6 703.8,2200.6 766.6,2147.9 826.8,2098.3 884.6,2051.7 940.1,2007.8 993.4,1966.6 1044.9,1927.7 1094.5,1891.2 1142.5,1856.7 1189.2,1824.1 1234.5,1793.3 1278.8,1764.2 1322.2,1736.6 1364.9,1710.3 1407.1,1685.3 1448.9,1661.4 1490.6,1638.5 1532.4,1616.5 1574.5,1595.2 1617.0,1574.7 1660.2,1554.7 1704.3,1535.2 1749.5,1516.1 1795.9,1497.3 1843.9,1478.8 1893.7,1460.4 1945.3,1442.2 1999.2,1424.0 2055.5,1405.8 2114.4,1387.6 2176.2,1369.2 2241.1,1350.8 2309.3,1332.1 2381.1,1313.3 2456.7,1294.3 2536.4,1275.0 2620.4,1255.5 2709.0,1235.7  A 1237.500,1237.500 0.00 0,1 2733.6,1401.5 L 2733.6,1401.5 2647.3,1409.0 2565.2,1417.2 2487.1,1426.1 2412.8,1435.6 2341.9,1445.8 2274.4,1456.6 2210.0,1468.1 2148.5,1480.1 2089.6,1492.7 2033.3,1505.9 1979.2,1519.8 1927.2,1534.3 1877.0,1549.4 1828.6,1565.3 1781.7,1581.9 1736.1,1599.3 1691.7,1617.6 1648.2,1636.7 1605.6,1656.9 1563.6,1678.1 1522.1,1700.5 1480.9,1724.1 1439.8,1749.0 1398.7,1775.3 1357.5,1803.2 1316.0,1832.8 1274.1,1864.1 1231.5,1897.4 1188.3,1932.7 1144.2,1970.2 1099.1,2010.1 1052.9,2052.5 1005.5,2097.5 956.7,2145.5 906.4,2196.4 854.6,2250.6 801.1,2308.2 745.8,2369.4 688.6,2434.4  Z" style="stroke-width: 1.0;  stroke: rgb(115,115,115); fill: rgb(0,69,41); opacity: 0.167;" />
</g>
<g id="track_0">
<path d="M 2098.668,2583.053 A 1237.500,1237.500 0.00 0,1 1946.8,2654.0 L 1946.8,2654.0 1938.3,2627.6 1931.9,2597.9 1927.7,2565.3 1925.5,2530.0 1925.3,2492.1 1927.2,2451.9 1931.0,2409.6 1936.7,2365.3 1944.2,2319.3 1953.6,2271.7 1964.6,2222.8 1977.4,2172.8 1991.7,2121.8 2007.6,2070.1 2025.0,2017.8 2043.8,1965.1 2063.9,1912.3 2085.3,1859.5 2107.9,1806.8 2131.5,1754.6 2156.2,1702.9 2181.8,1652.0 2208.3,1602.0 2235.5,1553.2 2263.3,1505.6 2291.7,1459.5 2320.6,1415.1 2349.8,1372.6 2379.2,1332.0 2408.8,1293.6 2438.4,1257.6 2468.0,1224.1 2497.3,1193.3 2526.3,1165.4 2554.9,1140.5 2582.9,1118.9 2610.2,1100.6 2636.6,1085.8 2662.2,1074.8  A 1237.500,1237.500 0.00 0,1 2709.0,1235.7 L 2709.0,1235.7 2685.3,1242.5 2660.5,1252.6 2634.9,1265.9 2608.5,1282.2 2581.5,1301.5 2554.1,1323.6 2526.4,1348.3 2498.6,1375.5 2470.7,1405.1 2442.9,1436.9 2415.4,1470.8 2388.2,1506.6 2361.5,1544.1 2335.4,1583.3 2310.0,1623.9 2285.3,1665.8 2261.6,1708.8 2238.8,1752.7 2217.1,1797.5 2196.6,1842.9 2177.2,1888.7 2159.2,1934.8 2142.5,1981.0 2127.3,2027.2 2113.6,2073.1 2101.4,2118.5 2090.7,2163.3 2081.8,2207.4 2074.5,2250.4 2068.9,2292.3 2065.0,2332.7 2062.9,2371.6 2062.6,2408.7 2064.1,2443.9 2067.4,2476.9 2072.5,2507.5 2079.4,2535.6 2088.1,2560.8 2098.7,2583.1  Z" style="stroke-width: 1.0;  stroke: rgb(115,115,115); fill: rgb(255,127,0); opacity: 0.667;" />
<path d="M 301.886,1313.774 A 1212.500,1212.500 0.00 0,1 1104.7,353.8 L 1104.7,353.8 1145.7,472.7 1183.6,582.0 1218.5,682.4 1250.5,774.3 1279.8,858.2 1306.6,934.6 1331.0,1004.1 1353.3,1067.2 1373.5,1124.3 1391.8,1176.1 1408.5,1223.0 1423.7,1265.5 1437.5,1304.2 1450.2,1339.6 1461.9,1372.1 1472.8,1402.3 1483.1,1430.8 1493.0,1457.9 1502.6,1484.2 1512.2,1510.3 1521.8,1536.7 1531.8,1563.8 1542.3,1592.1 1553.5,1622.3 1565.5,1654.7 1578.6,1689.9 1593.0,1728.3 1608.9,1770.6 1626.4,1817.2 1645.8,1868.6 1667.3,1925.3 1691.0,1987.8 1717.2,2056.7 1746.0,2132.4 1777.8,2215.4 1812.6,2306.2 1850.8,2405.4 1892.4,2513.4 1937.8,2630.7  A 1212.500,1212.500 0.00 0,1 705.0,2415.5 L 705.0,2415.5 739.0,2374.3 768.8,2334.2 794.6,2295.0 816.8,2256.6 835.6,2219.1 851.3,2182.3 864.1,2146.2 874.2,2110.7 881.8,2075.8 887.2,2041.4 890.4,2007.6 891.6,1974.2 891.0,1941.3 888.7,1908.9 884.8,1876.8 879.4,1845.2 872.5,1814.0 864.3,1783.3 854.7,1753.0 843.8,1723.2 831.6,1693.9 818.0,1665.1 803.1,1636.8 786.9,1609.2 769.2,1582.2 750.0,1555.9 729.2,1530.4 706.7,1505.7 682.4,1481.9 656.2,1459.1 627.9,1437.4 597.2,1416.8 564.2,1397.5 528.4,1379.5 489.8,1362.9 448.1,1348.0 403.1,1334.7 354.4,1323.3 301.9,1313.8  Z" style="stroke-width: 1.0;  stroke: rgb(115,115,115); fill: rgb(166,86,40); opacity: 0.667;" />
</g>
<defs>
<pattern id="hline" width="10" height="10" patternTransform="rotate(0 0 0)" patternUnits="userSpaceOnUse"><line x1="0" y1="0" x2="10" y2="0" style="stroke:black;stroke-width:5" /></pattern>
<pattern id="checker" width="10" height="10" patternTransform="rotate(45 0 0)" patternUnits="userSpaceOnUse"><line x1="0" y1="0" x2="10" y2="0" style="stroke:black;stroke-width:5" /><line x1="0" y1="0" x2="0" y2="10" style="stroke:black;stroke-width:4" /></pattern>
<pattern id="vline" width="10" height="10" patternTransform="rotate(0 0 0)" patternUnits="userSpaceOnUse"><line x1="0" y1="0" x2="0" y2="10" style="stroke:black;stroke-width:5" /></pattern>
<pattern id="hline-sparse" width="10" height="10" patternTransform="rotate(0 0 0)" patternUnits="userSpaceOnUse"><line x1="0" y1="0" x2="10" y2="0" style="stroke:black;stroke-width:2" /></pattern>
<pattern id="vline-sparse" width="10" height="10" patternTransform="rotate(0 0 0)" patternUnits="userSpaceOnUse"><line x1="0" y1="0" x2="0" y2="10" style="stroke:black;stroke-width:2" /></pattern>
<pattern id="checker" width="10" height="10" patternTransform="rotate(45 0 0)" patternUnits="userSpaceOnUse"><line x1="0" y1="0" x2="10" y2="0" style="stroke:black;stroke-width:2" /><line x1="0" y1="0" x2="0" y2="10" style="stroke:black;stroke-width:2" /></pattern>
</defs>
</svg>
