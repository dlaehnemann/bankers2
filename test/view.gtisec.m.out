# This file was produced by bcftools (1.2-185-g0d3a2f2-dirty+htslib-1.3-2-g6a745fe), the command line was:
# 	 bcftools gtisec  -m test/view.vcf
# Genotype intersections across samples:
@SMPS NA00003 NA00002 NA00001
# The first 3 lines contain the counts for missing values of each sample in the order provided
#   in the SMPS-line above. Intersection counts only start afterwards.
# Subset intersection counts are in global banker's sequence order.
#   After exclusive sample counts in order of the SMPS-line, banker's sequence continues with:
#   NA00003,NA00002   NA00003,NA00001   ...
# [1] Number of shared non-ref genotypes
4
1
1
9
4
5
1
0
8
1
